parlai/scripts/interactive.py

```diff
@@ -52,6 +52,10 @@ def interactive(opt, print_parser=None):
         opt = opt.parse_args()
     opt['task'] = 'parlai.agents.local_human.local_human:LocalHumanAgent'
 
+    if opt.get('eval_candidates', '') in ['inline', 'batch']:
+        print('[ Warning: using candidate type unavailable during interactive.'
+              ' Consider switching to "-ecands vocab" or "-ecands fixed". ]')
```

Comment:
nit i prefer spelling out long options in error messages `--eval-candidates` .

It seems weird to have this exception for rankers in interactive. should we not have the error message in TorchRankerAgent?